client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -369,7 +369,7 @@ class EmailProvidersComparison extends Component {
 					comment:
 						'{{fullPrice/}} is the formatted full price, e.g. $20; {{discountedPrice/}} is the discounted, formatted price, e.g. $10',
 			  } )
-			: translate( '{{price/}} /user /month billed annually', {
+			: translate( '{{price/}} /mailbox /month billed annually', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.